webapp/src/main/java/org/opentestsystem/rdw/admin/model/AppSettings.java

```diff
@@ -1,9 +1,17 @@
 package org.opentestsystem.rdw.admin.model;
 
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+@Component
 public class AppSettings {
 
+    @Value("${app.analytics.trackingId}")
     private String analyticsTrackingId;
 
+    @Value("${app.reporting.webapp.url}")
+    private String reportingWebappURL;
```

Comment:
I think this should be `reportingWebappUrl`. Whether i personally like it or not, the convention is to fully camel-case even for acronyms and initializations.